recipes/systemc-components/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, tools, CMake
+from conan.errors import ConanInvalidConfiguration
+import functools
+import os
+import shutil
+
+required_conan_version = ">=1.43.0"
+
+class SystemcComponentsConan(ConanFile):
+    name = "systemc-components"
+    description = """A light weight productivity library for SystemC and TLM 2.0"""
+    homepage = "https://minres.github.io/SystemC-Components"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+    topics = ("systemc", "modeling", "tlm", "scc")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "SC_WITH_PHASE_CALLBACKS": [True, False],
```

Comment:
```suggestion
        "with_phase_callbacks": [True, False],
```
Prefer lower-case always, Linux users will be thankful.